README.md

```diff
@@ -91,6 +91,11 @@ Japan:
   - [Rust of Us](https://rust-of-us.doorkeeper.jp/)
   - [Rust Tokyo](http://rust.connpass.com/)
 
+Malaysia:
+
+* Kuala Lumpur
+  - Meetup is too expensive, check [Rust Lang Malaysia](https://www.facebook.com/groups/1876280775927500/) or Telegram Group `¯\_(ツ)_/¯`
```

Comment:
No need to have Meetup :)
Just don't mention it and link to the facebook group directly.

Is there an invite link to Telegram or will folks need to ask for an invitation through facebook?

Reply:
The facebook link is that one, not sure if there's an invitation link, let me check.

Telegram link I will try to get it.

Reply:
I do saw the facebook link, I just don't think we need the phrase "Meetup is too expensive" in there.
The invitation link only referred to Telegram, no need for one for Facebook.

Reply:
Thanks for reminding, I updated the link.